scripts/battlefields/Riverne_Site_A01/ouryu_cometh.lua

```diff
@@ -0,0 +1,41 @@
+-----------------------------------
+-- Area: Riverne Site #A01
+-- Name: Ouryu Cometh
+-- !pos 732.550 -32.5 -506.544 30
+-----------------------------------
+local riverneID = zones[xi.zone.RIVERNE_SITE_A01]
+-----------------------------------
+
+local content = Battlefield:new({
+    zoneId        = xi.zone.RIVERNE_SITE_A01,
+    battlefieldId = xi.battlefield.id.OURYU_COMETH,
+    maxPlayers    = 18,
+    levelCap      = 75,
+    timeLimit     = utils.minutes(30),
+    index         = 0,
+    area          = 1,
+    entryNpc      = 'Unstable_Displacement',
+    exitNpc       = 'Spatial_Displacement',
```

Comment:
I'm not well versed on the new battlefield system. How exactly are these strings used? There are several NPC with both of these names, so will any of them trigger the battlefield functions? Do they need to be renamed to be distinguished from the other ones?

Reply:
I'm more of a fan of id definitions since it's more clear, but in certain cases it will add all

Reply:
For exit, it indeed applies to all

Reply:
> For exit, it indeed applies to all so if there are some that this does not apply to, it needs a unique name

Just so we're on the same page, is that out of scope for this PR? I'm happy to follow up with another to rename the exit NPC to something unique like Spatial_Displacement_BF (for battlefield)

Reply:
That is absolutely in scope for this PR.  You do _not_ want to assign the exit functions to NPCs where it does not apply.

Reply:
For what it's worth I tested this in a party so I could trigger the outside NPCs while having BCNM status and all the outside spatial displacements behaved normally and I only got the exit prompt after entering the battlefield. Still should distinguish them but good to know there seems to be some safety.

Reply:
> That is absolutely in scope for this PR. You do _not_ want to assign the exit functions to NPCs where it does not apply.

No problem, I've added a commit for that change as well.